### DISCUSSION TITLE
5/2(화) 제이미 - 세미뮤덕의 인생 뮤지컬 소개

### DISCUSSION THREAD

**Discussion by JJ503** [woowacourse-study/2023-presentation-study#18] [Q&A]
[[제이미] 세미뮤덕의 인생 뮤지컬.pdf](https://github.com/woowacourse-study/2023-presentation-study/files/11370765/default.pdf)

**Comment by yeonkkk** [comment]
Q1. `레드북`을 다음 인생 뮤지컬로 노리고 있는 이유는 무엇인가요?:)

**Reply by JJ503** [reply]
`레드북`은 한국 창작 뮤지컬로 이전에 제가 보았던 `여신님이 보고계셔`라는 작품의 창작자들이 만든 작품입니다.
여신님이 보고계셔라는 작품도 재밌게 보았기에 기대가 되기도 하고, 워낙 넘버가 좋기로 유명해 더 기대가 되는 작품입니다!
또한, 시놉시스를 보았을 때 보수적인 영국 빅토리아 시대에 본인이 하고 싶은 것을 하기 위해 맞서는 주인공 안나의 이야기로 넘버, 캐릭터, 줄거리 등 굉장히 사랑스러운 뮤지컬이라는 후기가 많아 많이 점점 궁금해지고 있습니다!

+) 보고 왔는데 정말 사랑스러운 뮤지컬이라고 생각합니다. 뮤지컬을 본 적은 없지만 궁금하신 분들께서 가볍게 보기 좋은 작품이라고 생각합니다!

**Comment by jyeost** [comment]
제이미의 뮤지컬 인생작 선택 기준은 무엇인가 궁금합니다 !
1. 줄거리
2. 넘버 
3. 메인 넘버의 반복과 변주가 많은 것
4. 라인업
5. 무대연출

**Reply by JJ503** [reply]
우선 뮤지컬에 애정이 생기는 과정은 줄거리와 넘버라고 생각합니다. (특히, 넘버가 머리에서 떠나지 못할 때가 많습니다 ㅎㅎ)
그리고 인생작이 된 뮤지컬들을 살펴보면 무대연출과 제 마음에 들어와 버린 캐릭터가 중요하게 작용했습니다.
개인적으로 무거운 이야기보다는 가벼운 이야기를 좋아해 대부분 동화같거나 사랑스러운 이야기들을 좋아하는 편입니다!
